{"uuid":"ae9740ee-5b27-45ca-ac87-1972dd365c39","historyId":"87292baf62c9e30cb5a898b3400715d5","fullName":"Demo feature: demo scenrio","labels":[{"name":"host","value":"WFQH2XPSQ2"},{"name":"thread","value":"3516@WFQH2XPSQ2.main(1)"},{"name":"feature","value":"Demo feature"},{"name":"story","value":"demo scenrio"},{"name":"suite","value":"Demo feature"},{"name":"testClass","value":"demo scenrio"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/pkhare/git/CucumberWithTestNG/src/test/resources/Features/Regression/Demo1.feature"},{"name":"package","value":"src.test.resources.Features.Regression.Demo1_feature.Demo feature"}],"links":[],"name":"demo scenrio","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"String in fieldName 'successful message' should be equal to 'contact added successfully'but displayed 'contact not added successfully' expected [contact added successfully] but found [contact not added successfully]","trace":"java.lang.AssertionError: String in fieldName 'successful message' should be equal to 'contact added successfully'but displayed 'contact not added successfully' expected [contact added successfully] but found [contact not added successfully]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat asserts.Assertions.expectedTextsAreEqual(Assertions.java:10)\r\n\tat demo.stepDefinitions.DemoStep.verifyValidationMessage(DemoStep.java:35)\r\n\tat ✽.Verify validation message(file:///C:/Users/pkhare/git/CucumberWithTestNG/src/test/resources/Features/Regression/Demo1.feature:5)\r\n"},"stage":"finished","steps":[{"name":"Given  Enter username","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[{"name":"Step Enter username","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1661188070881,"stop":1661188070882}],"attachments":[],"parameters":[],"start":1661188070846,"stop":1661188070886},{"name":"And  Enter Password","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1661188070886,"stop":1661188070887},{"name":"Then  Verify validation message","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"String in fieldName 'successful message' should be equal to 'contact added successfully'but displayed 'contact not added successfully' expected [contact added successfully] but found [contact not added successfully]","trace":"java.lang.AssertionError: String in fieldName 'successful message' should be equal to 'contact added successfully'but displayed 'contact not added successfully' expected [contact added successfully] but found [contact not added successfully]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat asserts.Assertions.expectedTextsAreEqual(Assertions.java:10)\r\n\tat demo.stepDefinitions.DemoStep.verifyValidationMessage(DemoStep.java:35)\r\n\tat ✽.Verify validation message(file:///C:/Users/pkhare/git/CucumberWithTestNG/src/test/resources/Features/Regression/Demo1.feature:5)\r\n"},"stage":"finished","steps":[{"name":"Check the 'successful message' field is equal 'contact added successfully'","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"String in fieldName 'successful message' should be equal to 'contact added successfully'but displayed 'contact not added successfully' expected [contact added successfully] but found [contact not added successfully]","trace":"java.lang.AssertionError: String in fieldName 'successful message' should be equal to 'contact added successfully'but displayed 'contact not added successfully' expected [contact added successfully] but found [contact not added successfully]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat asserts.Assertions.expectedTextsAreEqual(Assertions.java:10)\r\n\tat demo.stepDefinitions.DemoStep.verifyValidationMessage(DemoStep.java:35)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat io.cucumber.java.Invoker.doInvoke(Invoker.java:66)\r\n\tat io.cucumber.java.Invoker.invoke(Invoker.java:24)\r\n\tat io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)\r\n\tat io.cucumber.java.JavaStepDefinition.execute(JavaStepDefinition.java:29)\r\n\tat io.cucumber.core.runner.CoreStepDefinition.execute(CoreStepDefinition.java:66)\r\n\tat io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:63)\r\n\tat io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)\r\n\tat io.cucumber.core.runner.TestStep.executeStep(TestStep.java:85)\r\n\tat io.cucumber.core.runner.TestStep.run(TestStep.java:57)\r\n\tat io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)\r\n\tat io.cucumber.core.runner.TestCase.run(TestCase.java:84)\r\n\tat io.cucumber.core.runner.Runner.runPickle(Runner.java:75)\r\n\tat io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)\r\n\tat io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:129)\r\n\tat io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)\r\n\tat io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:129)\r\n\tat io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)\r\n\tat io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:806)\r\n\tat org.testng.TestRunner.run(TestRunner.java:601)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:433)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"fieldName","value":"successful message"},{"name":"actual","value":"contact not added successfully"},{"name":"expected","value":"contact added successfully"}],"start":1661188070891,"stop":1661188070908}],"attachments":[],"parameters":[],"start":1661188070887,"stop":1661188070913}],"attachments":[],"parameters":[],"start":1661188070818,"stop":1661188070919}